### PR TITLE
fix: remove unnecessary chain switch

### PR DESCRIPTION
**Pull request by Addminus**

### CODE REVIEW

File: src/executionFiles/bridges/bridge.execute.ts [hunk 1]
```diff
@@ -45,6 +63,7 @@ export class BridgeExecutionManager {
     if (!oldCrossProcess || !oldCrossProcess.txHash) {
       if (action.fromToken.address !== constants.AddressZero) {
         // Check Token Approval only if fromToken is not the native token => no approval needed in that case
+        await chainSwitchClosure()
```
Comment (mathiasmoeller) [inline src/executionFiles/bridges/bridge.execute.ts:66]: This will not stop execution here if `!updatedSigner`. Is this on purpose?

Comment (Addminus) [reply src/executionFiles/bridges/bridge.execute.ts:66]: the chainswitch does that

Comment (mathiasmoeller) [reply src/executionFiles/bridges/bridge.execute.ts:66]: How? It simply returns the execution, but nothing is done with that information as far as I can see?

**Comment by Addminus** [comment]
In fact the goal of this PR is not achievable. Closing it.

**Pull request closed without merging**